Source/common/SNTBlockMessage.m

```diff
@@ -122,6 +126,18 @@ + (NSURL *)eventDetailURLForEvent:(SNTStoredEvent *)event {
     formatStr = [formatStr stringByReplacingOccurrencesOfString:@"%machine_id%"
                                                      withString:config.machineID];
   }
+  if (hostname) {
```

Comment:
Use `hostname.length` here - that will ensure the value is both non-nil and not zero-length. Same for uuid and serial below

Reply:
Thanks, I've amended this. 👍